src/metabase/search/filter.clj

```diff
@@ -131,7 +128,8 @@
 
 (defn- default-created-at-filter-clause
   [model created-at]
-  (let [timezone-id         "UTC"
+  (let [;; we want the start/end date in UTC since all created_at columns are stored and displayed in UTC (not system tz)
```

Comment:
I don't think we can assume `created_at` is in UTC. There's nowhere we we enforce or recommend that the application database's time zone is UTC.

Reply:
I meant that mysql and postgresDB store timestamps with timezone in UTC internally.

Postgres
> For timestamp with time zone, the internally stored value is always in UTC (Universal Coordinated Time, traditionally known as Greenwich Mean Time, GMT). An input value that has an explicit time zone specified is converted to UTC using the appropriate offset for that time zone. If no time zone is stated in the input string, then it is assumed to be in the time zone indicated by the system's [TimeZone](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-TIMEZONE) parameter, and is converted to UTC using the offset for the timezone zone.

https://www.postgresql.org/docs/current/datatype-datetime.html

Mysql
> MySQL converts TIMESTAMP values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. (This does not occur for other types such as DATETIME.)

https://dev.mysql.com/doc/refman/8.0/en/datetime.html

Reply:
Sorry I think I had confused myself. `timezone-id` should be the browser timezone, not UTC, right? Since that is what the user thinks they are selecting the dates relative to.